docs/developers/ethereum-differences.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Ethereum and ConsenSys zkEVM differences
+description: Opcode differences between Ethereum and ConsenSys zkEVM
+sidebar_position: 3
+---
+
+# Ethereum and ConsenSys zkEVM differences
```

Comment:
Should we remove this? We currently don't have any differences listed.

Reply:
There are differences, but I'm waiting on Declan to provide them
i guess we should remove for now then

Reply:
Not sure we should remove it.
- short-term: it's a question everyone will have, it's important to address it
- long-term: we might have a bit more differences